languages/rust/exercises/concept/vec-macro/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+/// vec-macro for creating empty vectors, and vectors of particular size, with vec!
```

Comment:
Nit: I don't think that this line is strictly necessary; I'd prefer to remove it. If you prefer to keep it, that's fine, but then it should apply to module scope. As-is, `rustdoc` will generate a warning about a doc-comment which doesn't attach to anything.

To change it to apply to module scope, do this:

```suggestion
//! vec-macro for creating empty vectors, and vectors of particular size, with vec!
```

Reply:
Cool! thanks for the tip. I did not know this but see the value avoiding needless warnings.